api/test/integration/test_security_GET_endpoints.tavern.yaml

```diff
@@ -990,6 +990,19 @@ stages:
       status_code: 200
       verify_response_with:
         - function: tavern_utils:test_token_raw_format
+      save:
+        $ext:
+          function: tavern_utils:test_save_token_raw_format
```

Comment:
Related to the question above.

Reply:
Currently is our only option to save a raw token.
Another proposal:
* Test the raw token in another stage.
* Remove raw format from this test and parse the token from its json response (This could be done without using external functions).

But as we discussed this will remain as is since it's our best choice at the moment.